lib/kdump_utils.pm

```diff
@@ -164,4 +170,130 @@ sub do_kdump {
     script_run "echo c > /proc/sysrq-trigger", 0;
 }
 
+sub install_service {
+    zypper_call("in kdump");
+}
+
+# for kdump service we need use yast2 kdump to enable it
+# and reboot to make kdump service enabled and activated.
+sub enable_service {
+    activate_kdump();
```

Comment:
what is the point to take a function a and just wrap it in another function?

Reply:
OK, I'll remove this.